packages/frontend/components/vrf/RandomNFT/ExternalLink.tsx

```diff
@@ -29,11 +24,15 @@ export interface Props {
 export function ExternalLink({ tokenId }: Props): JSX.Element {
   const { chainId } = useEthers()
 
+  const contract = useContractConfig('RandomSVG')
+
   const active = chainId === OPENSEA_TESTNET
 
+  const url = active && `${OPENSEA_URL}/assets/${contract.address}/${tokenId}`
+
   return (
     <HStack>
-      <Link href={active && getUrl(tokenId)} isExternal>
+      <Link href={url} isExternal>
         See on OpenSea Testnet Marketplace <ExternalLinkIcon mx="2px" />
       </Link>
```

Comment:
There is the following warning "Warning: Received `false` for a non-boolean attribute `href`."

If is not active the url will be false and the link will be broken. Maybe if is not active there is no need to show the link?

```suggestion
      {active && <Link href={url} isExternal>
        See on OpenSea Testnet Marketplace <ExternalLinkIcon mx="2px" />
      </Link>}
```

Reply:
It's visible so if you're not on Rinkeby you can see it's available as an option.